syntax = "proto3";

message SizeModel {
    string size = 1;
    int32 price = 2;
    int32 stock = 3;
}

message ProductItemFormModel {
    string color = 1;
    string image_url = 2;
    repeated SizeModel sizes = 3;
}

message ProductFormModel {
    string product_name = 1;
    string product_desc = 2;
    string category_id = 3;
    repeated ProductItemFormModel product_item = 5;
}
